src/data/list/pairwise.lean

```diff
@@ -104,16 +104,24 @@ protected lemma pairwise.sublist : Π {l₁ l₂ : list α}, l₁ <+ l₂ → pa
 | ._ ._ (sublist.cons2 l₁ l₂ a s) (pairwise.cons i h) :=
   (h.sublist s).cons (ball.imp_left s.subset i)
 
-lemma pairwise.forall_of_forall (H : symmetric R) (H₁ : ∀ x ∈ l, R x x) (H₂ : l.pairwise R) :
+lemma pairwise.forall_of_forall_of_flip (h₁ : ∀ x ∈ l, R x x) (h₂ : l.pairwise R)
+  (h₃ : l.pairwise (flip R)) :
   ∀ ⦃x⦄, x ∈ l → ∀ ⦃y⦄, y ∈ l → R x y :=
 begin
-  induction l with a l IH, { exact forall_mem_nil _ },
-  cases forall_mem_cons.1 H₁ with H₁₁ H₁₂,
-  cases pairwise_cons.1 H₂ with H₂₁ H₂₂,
+  induction l with a l ih,
+  { exact λ x hx, hx.elim },
```

Comment:
I don't like this change: it abuses defeq of membership on the empty list and avoids the existing lemma we have for precisely this situation.